### PR TITLE
Sethnadu

### PR DESCRIPTION
**Pull request by sethnadu**

### CODE REVIEW

File: src/03_modules.py [hunk 1]
```diff
@@ -10,22 +10,25 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+for x in sys.argv:
```
Comment (lappjeff) [inline src/03_modules.py:13]: Never hurts to be more descriptive with your for loop variables! Ex `for arg in sys.argv`

File: src/06_tuples.py [hunk 2]
```diff
@@ -37,8 +37,9 @@ def dist(a, b):
 # YOUR CODE HERE
 
 t = (1, 2, 5, 7, 99)
-print_tuple(t)  # Prints 1 2 5 7 99, one per line
+for i in t:
```
Comment (lappjeff) [inline src/06_tuples.py:40]: Same as the previous comment, this could be better read as `for num in t`

File: src/07_slices.py [hunk 3]
```diff
@@ -12,16 +12,17 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print("Second Element:", a[1])
 
 # Output the second-to-last element: 9
-print()
+# print("With Len:", a[len(a)-2])
+print("Second to last:", a[4])
```
Comment (lappjeff) [inline src/07_slices.py:20]: You could also use pythons reverse operation to count backwards from the list, Ex: `a[-2:-1]`. This is a weird pattern to get used to for sure though

File: src/07_slices.py [hunk 4]
```diff
@@ -12,16 +12,17 @@
...
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print("Last Three: ", a[3:])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print("Two Middle: ", a[2:4])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
 print()
```
Comment (lappjeff) [inline src/07_slices.py]: Missed this one :).

**Review by lappjeff — CHANGES_REQUESTED** [review]
Give the last problem in `07_slices.py` a go and then you're good. All my comments are just nitpicking, this is great, functional code. Great job Seth!

File: src/07_slices.py (hunk 4 above)
Comment (sethnadu) [reply src/07_slices.py]: Thanks, I missed a few I saw, went back and fixed them

File: src/03_modules.py (hunk 1 above)
Comment (sethnadu) [reply src/03_modules.py:13]: Perfect thank you

**Review by lappjeff — APPROVED** [review]

**Comment by lappjeff** [comment]
Thanks Seth. Approved for day one!

**Pull request merged**